code/modules/clothing/glasses/glasses.dm

```diff
@@ -7,14 +7,14 @@
 	species_fit = list(VOX_SHAPED, GREY_SHAPED)
 
 /obj/item/clothing/glasses/science
-	name = "Science Goggles"
+	name = "science goggles"
 	desc = "nothing"
 	icon_state = "purple"
 	item_state = "glasses"
 	origin_tech = Tc_MATERIALS + "=1"
```

Comment:
@Kurfursten The plan here is to have the anomaly tech tree rely entirely on xenoarch. you want more science goggles? go find and deconstruct an alien bowl.